src/content/docs/extensions/delta.md

```diff
@@ -0,0 +1,145 @@
+:::
+
+### Copy the Delta table into a node table
+You can then use a `COPY FROM` statement to directly copy the contents of the Delta table into a Kùzu node table.
+
+```cypher
+CREATE NODE TABLE student (name STRING, ID INT64, PRIMARY KEY(ID));
+COPY student FROM '/tmp/student' (file_format='delta')
+```
+
+Just like above in `LOAD FROM`, the `file_format` parameter is mandatory when specifying the `COPY FROM` clause as well.
+
+```cypher
+// First, create the node table
+CREATE NODE TABLE student (name STRING, ID INT64, PRIMARY KEY(ID));
+```
+```
+┌─────────────────────────────────┐
+│ result                          │
+│ STRING                          │
+├─────────────────────────────────┤
+│ Table student has been created. │
+└─────────────────────────────────┘
+```
+```cypher
+COPY student FROM '/tmp/student' (file_format='delta');
+```
+```
+┌─────────────────────────────────────────────────┐
+│ result                                          │
+│ STRING                                          │
+├─────────────────────────────────────────────────┤
+│ 3 tuples have been copied to the student table. │
+└─────────────────────────────────────────────────┘
+```
+
+### Access Delta tables hosted on S3
+Kùzu also supports scanning/copying a Delta table hosted on S3 in the same way as from a local file system.
+Before reading and writing from S3, you have to configure the connection using the [CALL](https://kuzudb.com/docusaurus/cypher/configuration) statement.
+
+#### Supported options
+
+| Option name | Description |
+|----------|----------|
+| `s3_access_key_id` | S3 access key id |
+| `s3_secret_access_key` | S3 secret access key |
+| `s3_endpoint` | S3 endpoint |
+| `s3_url_style` | Uses [S3 url style](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html) (should either be vhost or path) |
+| `s3_region` | S3 region |
+
+#### Requirements on the S3 server API
+
+| Feature | Required S3 API features |
+|----------|----------|
+| Public file reads | HTTP Range request |
+| Private file reads | Secret key authentication|
+
+#### Scan Delta table from S3
+Reading or scanning a Delta table that's on S3 is as simple as reading from regular files:
+
+```sql
+LOAD FROM 's3://kuzu-sample/sample-delta' (file_format='delta')
+RETURN *
+```
+
+#### Copy Delta table hosted on S3 into a local node table
+
+Copying from Delta tables on S3 is also as simple as copying from regular files:
+
+```cypher
+CREATE NODE TABLE student (name STRING, ID INT64, PRIMARY KEY(ID));
+COPY student FROM 's3://kuzu-sample/student-delta' (file_format='delta')
+```
+
+## Limitations
+
+When using the Delta Lake extension in Kùzu, keep the following limitations in mind.
+
+- Writing (i.e., exporting to) Delta files is currently not supported.
+- We currently do not support scanning/copying nested data (i.e., of type `STRUCT`) in the Delta table columns.
```

Comment:
For delta we do support struct and other nested types because the underlying storage format is parquet.

Reply:
Got it, will make the change, thanks!